.circleci/config.yml

```diff
@@ -15,7 +15,7 @@ jobs:
     # You can specify an image from Dockerhub or use one of our Convenience Images from CircleCI's Developer Hub.
     # A list of available CircleCI Docker Convenience Images are available here: https://circleci.com/developer/images/image/cimg/node
     docker:
-      - image: cimg/openjdk:16.0.2-node
+      - image: cimg/python:3.8.8-node
```

Comment:
Someone please confirm if this is ok. I couldn't see anywhere JDK is being used. Python is needed for generating Glean files, but it can be installed manually if needed.

Reply:
The Firebase emulator is Java, as long as that works it's fine. I imagine that's why we used that one.